src/dune_file.ml

```diff
@@ -1103,20 +1104,21 @@ module Executables = struct
     end
     include T
 
-    let make mode kind =
+    let make mode kind loc =
       { mode
       ; kind
+      ; loc
       }
 
-    let exe           = make Best Exe
-    let object_       = make Best Object
-    let shared_object = make Best Shared_object
+    let exe           = make Best Exe Loc.none
+    let object_       = make Best Object Loc.none
+    let shared_object = make Best Shared_object Loc.none
```

Comment:
Maybe we should just change `make` to use `Loc.none`

Reply:
This was changed in 2ed578f to default to Loc.none.  Would you prefer it to be optional?  I think it is best in its current form unless there is some reason to have location information passed here which I was under the impression was not the case.

Reply:
It's good like this, thanks!